libs/langchain/langchain/document_loaders/web_base.py

```diff
@@ -74,12 +74,14 @@ def __init__(
                 "Received web_path and web_paths. Only one can be specified. "
                 "web_path is deprecated, web_paths should be used."
             )
-        if web_paths:
-            self.web_paths = list(web_paths)
+        if isinstance(web_path, str):
```

Comment:
there is a sep web_paths arg that is preferred to web_path

Reply:
oh my :)